### ISSUE TITLE
While the num_classes isn't 1000 , some error in sample.py!

### ISSUE THREAD

**Issue by Y1ZzLu** [facebookresearch/DiT#108]
Q1:when using single gpu train model ,` model.module` is not available 
Q2: sample.py `y_null = torch.tensor([1000] * n, device=device)` should be `y_null = torch.tensor([args.num_classes] * n, device=device)`
Q3:after training,i use my trained model to generate data,the sample.png looks like a gauss noise,my train loss is below 0.05! but i cant use this model !

**Comment by jizaza** [comment]
Q1:  you can just use `model` when using single gpu
Q2: In my opinion, `n` means the number of classes, `1000` maybe just a constant
Do you have any more ideas? Welcome to discuss!
Q3: you can change the `args.num-sampling-steps`

**Comment by Y1ZzLu** [comment]
@jizaza Thanks for your reply, I have solved the questions!

**Comment by racoonykc** [comment]
thanks!I found the same guassion noise

**Comment by Y1ZzLu** [comment]
@racoonykc By using a fine-tuning method to transfer learning from imagenet dataset to the target dataset, I use Lora fine-tuning provided by PEFT
